### ISSUE TITLE
Cross reference for figure/table

### ISSUE THREAD

**Issue by kozo2** [biohackrxiv/bhxiv-gen-pdf#28]
Quarto qmd has the ability to create cross references to figures and tables.

https://quarto.org/docs/authoring/cross-references.html

It would be nice if this could be realized in biohackrxiv.

**Comment by pjotrp** [comment]
This suggests you can use a label. Does that not work? https://tex.stackexchange.com/questions/268294/convert-existing-bibtex-to-bibjson

**Comment by egonw** [comment]
> This suggests you can use a label.

Yes, that works. I just added an example to the template: https://github.com/biohackrxiv/publication-template/commit/3d786124ef40410bf8ad7c7399b52e83cc68e337
